.gitmodules

```diff
@@ -24,7 +24,7 @@
 	url = git://github.com/opennetworklinux/ONLP-quanta
 [submodule "submodules/linux"]
 	path = submodules/linux
-	url = git://github.com/opennetworklinux/linux
+	url = git://github.com/InterfaceMasters/linux
```

Comment:
Yes, this is not required. This just to make out fork functional.

Vitaliy

Sent from my iPhone

> On Dec 21, 2015, at 20:39, Steven Noble <notifications@github.com> wrote:
> 
> In .gitmodules:
> 
> > @@ -24,7 +24,7 @@
> >  	url = git://github.com/opennetworklinux/ONLP-quanta
> >  [submodule "submodules/linux"]
> >  	path = submodules/linux
> > -	url = git://github.com/opennetworklinux/linux
> > +	url = git://github.com/InterfaceMasters/linux
> Hi Vitally,
> 
> This is not going to work, I assume if we merge your linux changes in, this will not be necessary.
> 
> —
> Reply to this email directly or view it on GitHub.
>